modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -89,7 +89,14 @@ You must provide these IP addresses to the installation program when you install
 [id="installation-vsphere-installer-infra-requirements-dns-records_{context}"]
 === DNS records
 
-You must create DNS records for two static IP addresses in the appropriate DNS server for the vCenter instance that hosts your {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
+You must create DNS records for the two reserved static IP addresses in the appropriate DNS server for the vCenter instance that hosts your {product-title} cluster.
+
+In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
+
+[NOTE]
+====
+When you install an {product-title} cluster on your vCenter that uses infrastructure that the installer provisions, the installation program uses its own internal load balancer based on HAProxy and keepalived. The installation program creates the HAProxy and keepalived pods in the `openshift-vsphere-infra` namespace.
```

Comment:
added info about IPI internal load balancer/HAProxy and keepalived per BZ

Reply:
It is better if  you updates your "docs.openshift.com" , as it is not mentioned or have the above info.
That cause a lot of confusion for any integration engineers who follows the online documents!.

Reply:
I agree, but there are [outstanding questions](https://bugzilla.redhat.com/show_bug.cgi?id=1864282#c15) on the BZ that I have not been able to resolve. If you are able to help explain what further detail is required, I would definitely like to get this finally resolved :slightly_smiling_face: